### PR TITLE
Fix handling of spaces in getVersionFomJar

### PR DESCRIPTION
**Pull request by frederikaalund**
See #27.

Note that this commit is untested.

### CODE REVIEW

File: src/main/kotlin/global/InstallationManager.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ object InstallationManager {
 
     /** Gets the version of the wurstscript.jar via cli */
     fun getVersionFomJar() {
-        val proc = Runtime.getRuntime().exec("java -jar " + compilerJar.toAbsolutePath() + " --version")
+        val proc = Runtime.getRuntime().exec("java -jar \"" + compilerJar.toAbsolutePath() + "\" --version")
```
Comment (peq) [inline src/main/kotlin/global/InstallationManager.kt]: Probably better to pass an array and let Java do it automatically:

    val proc = Runtime.getRuntime().exec(arrayOf("java", "-jar", InstallationManager.compilerJar.toAbsolutePath().toString(), "--version"))

**Comment by frederikaalund** [comment]
Thanks for the review, @peq. I agree completely. I've pushed the changes.

**Comment by Frotty** [comment]
you duplicated the proc, you need to remove the upper line to make the build succeed.
Thanks anyways for the PR, instead of merging into master I will most likely just integrate it into the next-gen branch which will be merged soon.

**Comment by frederikaalund** [comment]
> you duplicated the proc variable, you need to remove the line above to make the build succeed.

Would have been easier if I had just let you guys fix it in the first place. :) Anyways, I've removed the duplicate line. I don't know how that happened. Mental blunder.

> Thanks anyways for the PR, instead of merging into master I will most likely just integrate it into the next-gen branch which will be merged soon.

That sounds good with me. I just want #27 fixed.

**Comment by Frotty** [comment]
Thanks again @frederikaalund 🍰 

merged into #26

**Pull request closed without merging**